samples/bookinfo/platform/kube/bookinfo.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       serviceAccountName: bookinfo-details
       containers:
       - name: details
-        image: docker.io/istio/examples-bookinfo-details-v1:1.16.2
+        image: maistra/examples-bookinfo-details-v1:2.1.0
```

Comment:
@luksa is this file wrong? The images below are 2.0, this is 2.1

Reply:
I guess so. I've now changed all tags in bookinfo.yaml to 2.1.0 so they match those in the files with individual deployments (e.g. bookinfo-details.yaml, etc.). Thanks.

Reply:
Thanks :)